tests/E2E/scripts/combine-reports.py

```diff
@@ -49,7 +49,12 @@ def combine(self, report_dir):
           print('Parsing {}'.format(report))
           f = open(report, 'r')
           raw_data = f.read()
-          parsed_data = json.loads(raw_data)
+          try:
+            parsed_data = json.loads(raw_data)
+          except json.decoder.JSONDecodeError:
+            # Could not load json file
```

Comment:
Should'nt you store the number of JSON failures somewhere, in order to get the information ?

Reply:
Not needed because everything is in /tmp directory to prevent useless space usage.